Formula/operator-sdk.rb

```diff
@@ -53,13 +53,9 @@ def install
       assert_match stable.specs[:revision], version_output
     end
 
-    # Create a new, blank operator
-    system "#{bin}/operator-sdk", "new", "test", "--repo=github.com/example-inc/app-operator"
-
-    cd "test" do
-      # Add an example API resource. This exercises most of the various pieces
-      # of generation logic.
-      system "#{bin}/operator-sdk", "add", "api", "--api-version=app.example.com/v1alpha1", "--kind=AppService"
-    end
+    # Create an example AppService operator. This exercises most of the various pieces
+    # of generation logic.
+    args = ["--type=ansible", "--api-version=app.example.com/v1alpha1", "--kind=AppService"]
+    system "#{bin}/operator-sdk", "new", "test", *args
```

Comment:
anything that we can assert after the system call?

Reply:
I think it creates some files

Reply:
Added a check for requirements.yml

These files/directories are also created:

- build/
- deploy/
- molecule/
- roles/
- watches.yaml

Reply:
One of them is good enough. I doubt it'll be a packaging issue if it stops creating just one of these.